src/tools/rbd/action/Create.cc

```diff
@@ -7,6 +7,7 @@
 #include "common/errno.h"
 #include <iostream>
 #include <boost/program_options.hpp>
+#include "librbd/ImageCtx.h"
```

Comment:
We **try** to avoid including private API methods within rbd CLI -- what are you attempting to use from it?